insolar/jet/bits_test.go

```diff
@@ -17,11 +17,60 @@
 package jet
 
 import (
+	"fmt"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+// bitsToString converts byte slice to plain string representation in bits.
+func bitsToString(bitslice []byte) string {
```

Comment:
Why code has been moved to test?

Reply:
We discussed this method yesterday. It's being used only in tests. Because of the situation, linter shows warnings about `dead code`